src/App.js

```diff
@@ -1,8 +1,12 @@
 import './App.css';
-import Footer from '../src/components/Footer.js';
+import FeedCard from './components/FeedCard.js';
```

Comment:
please revert App.js back before committing. You can reset it and then commit again.